javascript/javascript2/week2/weather-wear.js

```diff
@@ -0,0 +1,11 @@
+function degrees (tempurature) {
+  let a = "Wear T-shirts"
+  let b = "Wear sweater"
+  if (tempurature > 18) {
+    return a
+  } else {
+    return b
+  }
+};
```

Comment:
Try to add most cases. Ex: if temperature is less than 10 degrees, if temperature is between 10 to 18 degrees and if temperature is greater than 18 degrees.